modules/strelka/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to the `strelka` module will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0] - 2020-06-03
+
+This release was authored by Helena Winata.
+
+
+<!-- TODO: Explain each important module design decision below. -->
+
+- `_strelka_run` outputs to a directory because strelka creates vcf files in `${STRELKA_ANALYSIS_PATH}/results/variants`
+- `_strelka_dispatch` is used to call outputs based on the `{pair_status}` wildcard since outputs files are automatically generate.
+    - somatic workflow outputs `somatic.snvs.vcf.gz` and `somatic.indels.vcf.gz`
+    -germline workflow outputs `variants.vcf.gz`
```

Comment:
Maybe to be explicit, we should use `germline.snvs.vcf.gz` instead of `variants.vcf.gz`. This way, the user knows what they're getting.

Reply:
`variants.vcf.gz` is automatically generated. Is there a way I can specify the output or I could change the name when symlinking to the output directory.